src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -113,6 +113,17 @@ public function __invoke(Node $node, CompilerContext $context): CompiledValue
                 return '';
             }
 
+            if (
+                $node instanceof Node\Expr\FuncCall
+                && $node->name instanceof Node\Name
+                && $node->name->toLowerString() === 'constant'
```

Comment:
Perhaps a silly edge case, so feel free to ignore, but `constant()` may be imported from a namespace?

Reply:
It should return FQN: https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Name.php#L113-L115